docs/features/decorators.rst

```diff
@@ -0,0 +1,89 @@
+Decorators
+==========
+
+This section explains Robottello decorators.
+
+.. contents::
+
+Modules
+-------
+
+Robottelo decorators are located under
+:doc:`decorator package </api/robottelo.decorators>`. Most of then are used to
+control if a test must be skipped or executed accordingly with specific
+configurations.
+
+stubbed
+-------
+
+``stubbed`` skips any test it decorates. A reason can be provided as parameter
+and its default is "Not Implemented". Most of times it is used to define all
+test cases regarding a feature. Example::
+
+    from robottelo.decorators import stubbed
+
+    @stubbed()
+    def test_negative_create_matcher_attribute_priority(self):
+        """Test will be implemented later"""
+
+    @stubbed('Some other reason than not implemented')
+    def test_positive_create_matcher_attribute_priority(self):
+        """Test will be implemented later"""
+
+skip_if_bug_open
+----------------
+
+``skip_if_bug_open`` skips test based on bug tracker status. Currently Bugzilla
+and Redmine are supported. The decorator receives two parameters. The first
+indicates the tracker and must be onf of 'bugzilla' or 'redmine'. The second
+parameter indicates the bug id on respective bug tracker. Example::
+
+    from robottelo.decorators import skip_if_bug_open
+
+    @skip_if_bug_open('bugzilla', 1297308)
+    def test_negative_add_puppet_content(self):
+        """Attempt to associate puppet repos within a custom content
+        view directly
+        """
+
+    @skip_if_bug_open('redmine', 23456789)
+    def test_positive_add_puppet_content(self):
+        """Attempt to associate puppet repos within a custom content
+        view directly
+        """
+
+Decorator communicates with bug tracker to know the status of respective bug
+id. If it is still open, test will be skipped. Once dev team fix and change bug
```

Comment:
Once bz is fixed and moved to `ON_QA` or `Verified` state, test will automatically be run with next builds. ?